### PR TITLE
Set Isolated/Internal properties of target feed configs

### PR DESCRIPTION
**Pull request by JohnTortugo**

### CODE REVIEW

**Comment by JohnTortugo** [comment]
@mmitche as per our discussion a few days ago we can consider (for now?) feeds for installers/checksums as isolated ones, right?

File: src/Microsoft.DotNet.Arcade.Sdk/tools/SdkTasks/SetupTargetFeeds.proj [hunk 1]
```diff
@@ -155,20 +163,26 @@
         AssetSelection="NonShippingOnly"
         Include="NetCore"
         TargetURL="$(AzureDevOpsStaticTransportFeed)"
+        Internal="false"
+        Isolated="true"
```
Comment (JohnTortugo) [inline src/Microsoft.DotNet.Arcade.Sdk/tools/SdkTasks/SetupTargetFeeds.proj]: @mmitche the value of `AzureDevOpsStaticTransportFeed` is `'https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json'`. That is not an Isolated feed, right? If that is true, which feed should be used for non-shipping packages on stable builds?

Comment (mmitche) [reply src/Microsoft.DotNet.Arcade.Sdk/tools/SdkTasks/SetupTargetFeeds.proj]: This should be set to false, but it probably doesn't matter (we don't push things to this feed that are stabilized).

File: src/Microsoft.DotNet.Arcade.Sdk/tools/SdkTasks/SetupTargetFeeds.proj [hunk 2]
```diff
@@ -182,18 +196,24 @@
       <TargetFeedConfig
         Include="NetCore;OSX;Deb;Rpm;Node;BinaryLayout;Installer;Checksum;Maven;VSIX"
         TargetURL="$(StaticInternalFeed)"
+        Internal="true"
+        Isolated="false"
         Type="AzDoNugetFeed"
         Token="$(AzdoTargetFeedPAT)" />
       
       <TargetFeedConfig
         Include="OSX;Deb;Rpm;Node;BinaryLayout;Installer;Maven;VSIX"
```
Comment (riarenas) [inline src/Microsoft.DotNet.Arcade.Sdk/tools/SdkTasks/SetupTargetFeeds.proj]:
```suggestion
        Include="OSX;Deb;Rpm;Node;BinaryLayout;Installer;Maven;VSIX;Badge"
```

You're not touching that line, but I just noticed the mismatch

**Pull request merged**